cpp_utils/include/cpp_utils/utils.hpp

```diff
@@ -177,6 +177,34 @@ CPP_UTILS_DllAPI void* copy_to_void_ptr(
         const T* source,
         size_t size = sizeof(T));
 
+CPP_UTILS_DllAPI bool replace_first(
+    std::string& st,
+    std::string const& to_replace,
+    std::string const& replace_by);
+
+CPP_UTILS_DllAPI unsigned int replace_all(
+    std::string& st,
+    std::string const& to_replace,
+    std::string const& replace_by);
+
+/**
+ * @brief Remove undesired substrings from string.
+ *
+ * Characters as \r or not UTF-8 could be removed or replaced by a string.
+ * By default, it will replace "\n" and \r by an empty string (= remove it).
```

Comment:
What do you mean with "="?

```suggestion
 * By default, it will replace "\n" and "\r" by an empty string (= remove it).
```